docs/ReleaseNotes.md

```diff
@@ -1,6 +1,15 @@
 Release Notes for Version 2
 ============================
 
+Build 008
+-------
+
+Published as version 2.4.1
```

Comment:
When adding a new feature, the minor version number should be increased. In this case, 2.5.0

Reply:
Yes, you're right. I update it.